src/hooks/usePostSubmissionAction.tsx

```diff
@@ -2,17 +2,20 @@ import { useEffect, useState } from 'react';
 import { PostSubmissionAction } from '../api/types';
 import { getPostSubmissionActionById } from '../registry/registry';
 
-export function usePostSubmissionAction(actionIds: Array<string>) {
-  const [actions, setActions] = useState<Array<PostSubmissionAction>>([]);
+export function usePostSubmissionAction(actionRefs: Array<{ actionId: string; config?: Record<string, any> }>) {
```

Comment:
what impact does this change have to already existing post submission actions

Reply:
The signature of the function has changed but still supports strings. See: https://github.com/openmrs/openmrs-form-engine-lib/blob/4e0f809b1ac5fabf039aa70d41b18e69b5b86cec/src/hooks/usePostSubmissionAction.tsx#L11